solidity/contracts/bridge/BridgeState.sol

```diff
@@ -133,6 +135,9 @@ library BridgeState {
         // `redemptionTreasuryFeeDivisor` and `redemptionTxMaxFee`
         // parameters in order to make requests that can incur the
         // treasury and transaction fee and still satisfy the redeemer.
+        // Additionally, the value of this parameter must always be greater
+        // than `movingFundsDustThreshold` in order to prevent redemption
+        // requests with values smaller or equal to `movingFundsDustThreshold`.
```

Comment:
```suggestion
        // requests with values lower or equal to `movingFundsDustThreshold`.
```

Reply:
Done in https://github.com/keep-network/tbtc-v2/pull/241/commits/f2a5d262f26294ddcbdaf4295aa33b3cc18a5418.